pkg/env/env.go

```diff
@@ -306,7 +310,9 @@ func (e *testEnv) getFinishActions() []action {
 
 func (e *testEnv) execFeature(ctx context.Context, t *testing.T, f types.Feature) context.Context {
 	featName := f.Name()
-
+	if featName == "" {
+		featName = fmt.Sprintf("%s-feature-%x", t.Name(), e.rnd.Int63())
```

Comment:
The example is resulting in an output that looks like this:

```
TestTableDriven/TestTableDriven-feature-8e14a2df72a380a/A_simple_feature
```

I would drop the `t.Name()` to save a few repeated characters from the output:

```suggestion
		featName = fmt.Sprintf("feature-%x",  e.rnd.Int63())
```

Reply:
Good idea.